ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthPeer.java

```diff
@@ -411,11 +446,12 @@ public void registerHeight(final Hash blockHash, final long height) {
   }
 
   public int outstandingRequests() {
-    return headersRequestManager.outstandingRequests()
-        + bodiesRequestManager.outstandingRequests()
-        + receiptsRequestManager.outstandingRequests()
-        + nodeDataRequestManager.outstandingRequests()
-        + pooledTransactionsRequestManager.outstandingRequests();
+    final AtomicInteger count = new AtomicInteger(0);
```

Comment:
Cool. Did you run into a concurrency issue that prompted this?

Reply:
No it is rather the solution used to be able to increment an integer using streams